### PR TITLE
Add Autometa

### PR DESCRIPTION
**Pull request by WiscEvan**
Add Autometa tool - https://github.com/KwanLab/Autometa

- [x] source url is stable
- [x] sha256 included
- [x] build number is 0
- [x] `.bat` file is removed
- [x] Unnecessary comments removed
- [x] Entrypoint and import tests included
- [x] Following FSH
- [x] License included
- [x] Package does not exist in `defaults` or `conda-forge`
- [x] Package is related to handling biological sequences
- [x] No custom wrapper scripts installed
- [ ] `noarch` specification under `build:`
- [ ] Patching details

----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

File: recipes/autometa/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/KwanLab/Autometa/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: generic
```
Comment (dpryan79) [inline recipes/autometa/meta.yaml:15]: Can you change this to "noarch: python"? It'll be handled better then.

File: recipes/autometa/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+  script:
+    - {{ PYTHON }} -m pip install . --ignore-installed --no-deps -vv
+    - {{ PYTHON }} -m pip install git+https://github.com/eamid/trimap --ignore-installed -vv
+    - {{ PYTHON }} setup.py install --single-version-externally-managed --record=record.txt
```
Comment (dpryan79) [inline recipes/autometa/meta.yaml:19]: Can you remove the last two lines here? We just need line 17.

**Review by dpryan79 — CHANGES_REQUESTED** [review]
Just a few changes and this should be good to go.

**Comment by WiscEvan** [comment]
@BiocondaBot please add label

**Pull request merged**